src/pycram/designators/actions/actions.py

```diff
@@ -32,14 +40,24 @@ def perform(self) -> None:
         """
         pass
 
-    @abc.abstractmethod
     def to_sql(self) -> Action:
         """
```

Comment:
add some description in the doc string on what requirements are needed to be able to do this automatically. And also when and how to overwrite it